tools/deployment/package/docker-compose.base.yaml

```diff
@@ -331,18 +342,21 @@ services:
   garbage-collector:
     <<: *service_defaults
     hostname: "garbage_collector"
+    deploy:
+      # Value must be either 0 or 1. Set to 0 to disable the garbage collector.
+      replicas: "${CLP_GARBAGE_COLLECTOR_ENABLED:-1}"
     environment:
-      CLP_DB_PASS: "${CLP_DB_PASS}"
-      CLP_DB_USER: "${CLP_DB_USER}"
+      CLP_DB_PASS: "${CLP_DB_PASS:?error}"
+      CLP_DB_USER: "${CLP_DB_USER:?error}"
       CLP_HOME: "/opt/clp"
       CLP_LOGGING_LEVEL: "${CLP_GC_LOGGING_LEVEL:-INFO}"
       CLP_LOGS_DIR: "/var/log/garbage_collector"
       PYTHONPATH: "/opt/clp/lib/python3/site-packages"
     volumes:
```

Comment:
Don't we also need the AWS config directory so the collector can delete archives from S3?

Reply:
right. added and validated